app/grandchallenge/core/widgets.py

```diff
@@ -102,7 +102,7 @@ class MarkdownEditorFullPageWidget(MarkdownEditorInlineWidget):
     template_name = "markdownx/full_page_widget.html"
 
     def __init__(self, attrs=None):
-        default_attrs = {"rows": "40"}
+        default_attrs = {"rows": "30"}
```

Comment:
Does this need to be set or will the widget take up as much space as possible?

Reply:
It will not take up as much space as possible. I set this as an initial size, but the user can resize the widget. The reason the widget has a fixed size is so that the user can have two independent scroll areas to line up the markdown with the preview content. (Scrolling is not linked btw.) 
If we let the widget take up the space it needs, the user can only use the scroll bar of the page. This means that if the markdown does not line up with the preview content, they still need to scroll up and down to edit and preview.